app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/manager/TotpCodeManagerImpl.kt

```diff
@@ -28,6 +29,9 @@ class TotpCodeManagerImpl @Inject constructor(
     override fun getTotpCodesFlow(
         itemList: List<AuthenticatorItem>,
     ): Flow<List<VerificationCodeItem>> {
+        if (itemList.isEmpty()) {
```

Comment:
This is one of two changes that is technically unrelated to this PR. 

I had a bug here where when mapping an empty list, I got a flow that never emits. This change instead maps to a flow that just emits empty list, which gets rendered correctly as the empty state.

If you'd like me to pull this into a separate PR quick, I'd be happy to!

Reply:
Nice catch! I think it would be good to extract this fix and include a test for it if we can.

Reply:
https://github.com/bitwarden/authenticator-android/pull/231 🤗